scripts/send-email-to-unresolved-breach-subscribers.js

```diff
@@ -0,0 +1,23 @@
+const DB = require('../db/DB')
+const EmailUtils = require('../email-utils')
+const { LocaleUtils } = require('../locale-utils')
+
+async function sendUnresolvedBreachEmails () {
+  const subscribers = await DB.getSubscribersWithUnresolvedBreaches()
+
+  subscribers.forEach(async subscriber => {
+    const supportedLocales = [subscriber.signup_language, 'en'].filter(Boolean) // filter potential nullish signup_language
```

Comment:
question (non-blocking): so we're sending this to everyone who has `en` in their signup language at all. Even if it's their 2nd or 3rd language?

Reply:
The way this is intended to work is to use the subscriber's signup language first.  If for some reason that's null (mine is locally), we still have `en` as the fallback.